netbox/ipam/views.py

```diff
@@ -216,6 +216,7 @@ class ASNRangeASNsView(generic.ObjectChildrenView):
     child_model = ASN
     table = tables.ASNTable
     filterset = filtersets.ASNFilterSet
+    template_name = 'generic/object_tab.html'
```

Comment:
Is there a need to define the template again? The same is already defined on `ObjectChildrenView`

Reply:
I've removed it from the base class per my comment over at https://github.com/netbox-community/netbox/pull/13388#discussion_r1285898157. I'm probably being overly cautious but I want to be very careful to not introduce a breaking change for any plugins using the view.